Documentation/debugging/unix-instructions.md

```diff
@@ -33,7 +33,6 @@ It is also possible to debug .NET Core crash dumps using lldb and SOS. In order
 - Matching coreclr/corefx runtime bits from the crash. To get these, you should either:
   - Download the matching Jenkins archive onto your repro machine.
   - Check out the coreclr and corefx repositories at the appropriate commit and re-build the necessary portions.
-  - You can also download the matching "symbols" nuget package from myget.org. There is a "Download Symbols" button in the myget UI for this purpose.
```

Comment:
What's the symbols story now -- do they all go to https://msdl.microsoft.com/download/symbols automatically? Or is there an equivalent method?

Reply:
Correct they al go to msdl.